packages/xo-server/src/xo-mixins/proxies.mjs

```diff
@@ -57,6 +57,17 @@ function addProxyUrl(proxy) {
   proxy.url = url.href
 }
 
+async function getProxyVersion(proxy) {
```

Comment:
`getProxyVersion` we imagine that we will retrieve the version of the proxy
```js
const proxyVersion = getProxyVersion(proxy)
```
But here, that not the case. `addProxyVersion` would fit more of what is doing the function.